app/components/sidebars/settings/settings_sidebar.ios.js

```diff
@@ -27,6 +27,8 @@ export default class SettingsDrawer extends SettingsSidebarBase {
         this.state = {
             deviceWidth: Dimensions.get('window').width,
             openDrawerOffset: DRAWER_INITIAL_OFFSET,
+            showStatus: true,
+            showRetryMessage: false,
```

Comment:
shouldn't this be based on super? in any way `showStatus` should not default to true as the CustomStatus feature could be disabled or not present

Reply:
We cannot access `super.state` here. That is showing `undefined` here. Is there any other way to do this?